src/main/java/ua/softserveinc/tc/controller/admin/DayOffController.java

```diff
@@ -64,6 +68,7 @@
         if (dayOffService.dayOffExist(dayOff.getName(), dayOff.getStartDate())) {
             log.warn("There is another day off with the same name: " + dayOff.getName() + ", or" +
                     "with the same start date: " + dayOff.getStartDate());
+            return new ResponseEntity<>(HttpStatus.NOT_FOUND);
```

Comment:
Sorry, but you are wrong. Look again. The logic was absolutely another. You change it. I doesn't see it before. When it was agreed with PM? In the past method create this day of and only produces warning, but now it not create that dayoff but returns HttpStatus.NOT_FOUND